scipy/stats/tests/test_binned_statistic.py

```diff
@@ -39,6 +40,15 @@ def test_gh5927(self):
         for statistic in statistics:
             res = binned_statistic(x, v, statistic, bins=10)
 
+    def test_big_number_std(self):
```

Comment:
might want to move the `# for gh-10126` down to the test proper, now that I think about it

Reply:
> might want to move the `# for gh-10126` down to the test proper, now that I think about it

Done